cmd/run.go

```diff
@@ -60,11 +60,11 @@ var runCommand = &cobra.Command{
 				volume = user.HomeDir + volume[1:]
 			}
 			dockerArgs = append(dockerArgs, "-v")
-			dockerArgs = append(dockerArgs, volume)
+			dockerArgs = append(dockerArgs, os.ExpandEnv(volume))
 		}
```

Comment:
I saw #54 , I quite like it as well
I wonder if there could not be several options:
- White list several environment variables, freeze the others
- Add an install option of --[no-]network option to run the commands with a --net None
- Notify the user when environment variables will be expanded (after  #54 merge)  ?